local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -20,6 +20,7 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"syscall"
```

Comment:
I think this pkg is deprecated, and we're supposed to use https://godoc.org/golang.org/x/sys/unix instead.